frontend/package.json

```diff
@@ -69,8 +69,10 @@
     "brace": "0.11.x",
     "classnames": "2.x",
     "core-js": "2.x",
+    "d3": "^5.9.2",
     "file-saver": "1.3.x",
     "font-awesome": "4.7.x",
+    "formik": "2.0.1-rc.1",
```

Comment:
We're locking to a prerelease version?

Reply:
@spadgett We are using V2 because it comes with support for custom formik hooks and it makes creating custom components wrapped around formik context much easier. They're actually at 2.0.1-rc..5 but because of a [bug](https://github.com/jaredpalmer/formik/issues/1560) related to validations i had to downgrade to rc-1. I'll keep track of it and update the version to latest as soon as there's fix.